tomviz/CropOperator.cxx

```diff
@@ -0,0 +1,127 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "CropOperator.h"
+
+#include "CropWidget.h"
+#include "vtkDataObject.h"
+#include "vtkExtractVOI.h"
+#include "vtkImageData.h"
+#include "vtkNew.h"
+
+namespace tomviz
+{
+
+CropOperator::CropOperator(const int *dataExtent, const double *dataOrigin,
+                           const double *dataSpacing, QObject* prnt)
+  : Superclass(prnt)
+{
+  for (int i = 0; i < 6; ++i)
+    {
```

Comment:
Let's stick with modern indentation, we are using the standard C++ indentation scheme and it is inconsistently applied in this file.